client/mysqlimport.c

```diff
@@ -410,14 +410,12 @@ static void lock_table(MYSQL *mysql, int tablecount, char **raw_tablename)
 
 
 
-static MYSQL *db_connect(char *host, char *database,
-                         char *user, char *passwd)
+static void db_connect(char *host, char *database,
```

Comment:
nit - since you are changing db_connect anyway.. can we have each of the `char*` parameters be changed to `const char*`?